guides/common/modules/proc_managing-host-packages.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Managing-Host-Packages_{context}"]
+= Managing Host Packages
+
+You can manage the packages of hosts even if the subscriptions are not attached to access repositories.
+
+.Procedure
+. Navigate to *Hosts > All Hosts* and select any Host.
+. Click on the *Content* tab and go to *Packages* where you can see the list of all the available packages with their status.
+
+The Packages pages are paginated, use the arrows to navigate through the pages.
```

Comment:
A lot of pages are paginated; I have never seen this note before. I am unsure if we want to add this to our docs. cc @Lennonka What do you think?

Reply:
I don't think we should include this.